src/main/java/com/carrotcreative/recyclercore/widget/ProgressRecyclerViewLayout.java

```diff
@@ -117,4 +124,134 @@ public void stopScroll()
     {
         mRecyclerView.stopScroll();
     }
+
+    // ========================================== //
+    // =============  Error State =============== //
+    // ========================================== //
+
+    public boolean isErrorStateEnabled()
+    {
+        if(mErrorStateView != null && mErrorStateView.getVisibility() == View.VISIBLE)
+        {
+            return true;
+        }
+        return false;
+    }
+
+    public void setErrorStateEnabled(boolean enable)
```

Comment:
You don't have to set the `setErrorView` before you call this method `isErrorStateEnabled `